Frontend/ui-customer/src/Screens/CustomerUtils/AddNewAddressScreen/ScreenData.purs

```diff
@@ -51,6 +51,9 @@ initData = {
     , distance : Nothing
     , showDistance : Just false
     , actualDistance : 0
+      , frequencyCount : Just 0
```

Comment:
if it's a maybe type, then why not this is `Nothing` and `Just 0` ??

Reply:
Updated